canonical/sphinx-docs-starter-pack
# Exit 1 on failure

Causes a non-zero exit status when there are woke findings. This in turn will fix the issue with failure reporting on the documentation GitHub workflow: https://github.com/canonical/documentation-workflows/issues/5#issuecomment-1744544831.

To test:

1. Checkout `main` branch
2. Edit some file to include a disallowed word (e.g. edit `readme.rst` by adding "master")
3. Run `make woke`
4. Observe that there is a "finding" reported by woke, warning of the use of the disallowed word
5. Run `echo $?` immediately afterwards - it should output `0`, indicating successful exit status

Repeat the above steps again on this branch. Notice that the command exits with non-zero exit status this time.
